### DISCUSSION TITLE
性能优化

### DISCUSSION THREAD

**Discussion by 1004834495** [ant-design/ant-design#50273] [General]
antd的Checkbox.Group中渲染过多的Checkbox导致页面卡顿，如何优化

**Comment by dislido** [comment]
最简单的方式是用支持虚拟滚动的Tree来代替Checkbox，但样式就只能是单列滚动的选项列表了

**Reply by rookie-luochao** [reply]
如果抽离了单独的渲染组件，并且使用了key无解的话，你必须变换实现思路了
比如改成multi select，先排序展示一部分，另外的通过搜索来进行选择，选中的就用tag标识出来
